quilt/tools/command.py

```diff
@@ -372,41 +381,92 @@ def _push_package(dry_run=False):
             }
         )
 
-    _push_package(dry_run=True)
+    print("Fetching upload URLs from the registry...")
```

Comment:
I specifically requested this because it can take a long time. The cli appears to hang. If streaming solves tis problem I guess we can get rid of it. Otherwise leave it in.